conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderAuxiliarySettersUtils.java

```diff
@@ -66,7 +100,7 @@ public static MethodSpec.Builder createCollectionSetterBuilder(
                         field.name));
     }
 
-    public static MethodSpec.Builder createOptionalSetterBuilder(
```

Comment:
I can extract this into a separate PR, but I noticed that we were overly exposing these methods which in turn was overly exposing the `EnrichedField` type.